language-family/.github/workflows/create-release-issue.yml

```diff
@@ -0,0 +1,37 @@
+name: Create reminder issue for buildpack releases
+
+on:
+  schedule:
+    - cron: '0 10 * * 3 '
```

Comment:
This opens the issues on wednesday. Should we open the issues on monday instead? This would give maintainers the largest window possible to ship before end of week.

Also it means the issue will say: "Week of XX MM" where XX will be the date of a monday, which I think is slightly better than the date of wednesday of a given week.

Reply:
I kind of like Wednesday. Coming in on Monday we usually have tons of issues to look at and organize. It's kind of nice to not have these come in until Wednesday

Reply:
I agree that the `Week of MM XX` being a Wednesday is slightly strange though

Reply:
> My thought is that its kind of nice to not have these come in until Wednesday

Sure, I get that.

I don't agree - I'd prefer everything to come in Monday and let maintainers triage on their own pace - but I also don't feel strongly at all about this.

Regardless, I'd rather see this get merged and the automation start working.